src/Descriptors/Schema/Schema.php

```diff
@@ -402,12 +405,17 @@ protected function attributes(
 
               $schema = $fieldDataType->title($field->name());
 
-              $column = $field instanceof Attribute ? $field->column() : $field->name();
-              if (isset($example[$column])) {
-                  $schema = $schema->example($example[$column]);
-              }
-              if ($field->isReadOnly(null)) {
-                  $schema = $schema->readOnly(true);
```

Comment:
Why did you remove the read-only option?

Reply:
That was a mistake on my end. I have replaced it and added a better check than before